# command constants:

string PLAY=play
string STOP=stop
string PAUSE=pause
string UNPAUSE=unpause
string VOLUME_UP=volume_up
string VOLUME_DOWN=volume_down

string command
string[] args
